stable/rabbitmq-ha/values.yaml

```diff
@@ -280,7 +280,9 @@ prometheus:
       pullPolicy: IfNotPresent
 
     ## Port Prometheus scrapes for metrics
-    port: 9090
+    port: 9419
```

Comment:
Why did you change the port?

Reply:
@unguiculus The default port in the rabbitmq exporter docker image is 9090 (Which is why it was the default port) but, It'll change and, the default port will be 9419.
It might work with any of them (As the environment variable is being set in the change) but I think it might be better to reflect the true defaults. Should I set it back to 9090?

Reply:
I'd stick with the default to avoid any confusion.

Reply:
I'd suggest you set it back. Otherwise this might cause confusion.

Reply:
@unguiculus done